webpack.config.js

```diff
@@ -26,10 +73,7 @@ module.exports = {
         use: ["babel-loader", "source-map-loader"],
         exclude: /node_modules/,
       },
-      {
-        test: /\.s?css$/,
-        loader: ["style-loader", "css-loader", "sass-loader"],
-      },
+      styleLoaders,
       {
         test: /\.(png|jpe?g|svg|ico)$/,
```

Comment:
maybe a few things missing for the list used in the jest config  `(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)`?